CONTRIBUTING.md

```diff
@@ -71,6 +71,23 @@ when the PR is merged.
 Most PRs will be reviewed by two [approvers][prow-approvers] (listed [here](OWNERS)).
 Some maintainers add themselves to [`CODEOWNERS`](.github/CODEOWNERS) to manage their [review notifications][code-owners], but those entries have no governance significance.
 
+### Dependencies management
+
+Make sure [`vndr`](https://github.com/LK4D4/vndr) is installed.
+
+In order to add a new dependency to this project:
+
+- add a new line to `vendor.conf` according to `vndr` rules (e.g. `github.com/pkg/errors master`)
+- run `make PKG=github.com/pkg/errors vendor`
```

Comment:
I fear the `PKG` variable will be seen as an invite to update only **some** dependencies but not **all**.

I prefer to not expose the variable and always run `make vendor` which runs `vndr -whitelist` on the entire `vendor.conf`. In theory, only the altered dependencies will be updated (assuming non are pulling from a branch). If  others are updated as well, then there's something wrong in the `vendor.conf` but we're more likely to catch those.

Reply:
Sounds great. I am cool with having this a temporary fix!